mdpow/tests/test_runinput.py

```diff
@@ -85,8 +89,9 @@ class TestAlteredConfig(object):
 
     def setup(self):
     # load default bundled configuration
+        self.old_path = os.getcwd()
         self.cfg = mdpow.config.get_configuration(
-            os.path.join('mdpow', 'tests', 'testing_resources',
+            os.path.join(self.old_path, 'mdpow', 'tests', 'testing_resources',
                          'test_configurations', 'altered_runinput.yml'))
```

Comment:
Why do you need to put old_path there?  --- the 2.7 tests at least ran fine without this change. Remove `self.old_path`, should not be needed.

Reply:
Turns out it wasn't necessary, probably another thing I added trying to figure out some error.